twisher/src/Posts.js

```diff
@@ -0,0 +1,253 @@
+            author: 'Пестова Анна',
+            likes: ['Бегун Юлий', 'Иван Иванов'],
+        },
+        {
+            id: '14',
+            description: 'и при этом некоторые преподаватели любят использовать фразы *ну вы дома ичего не делаете, так что вот вам еще задание*',
+            createdAt: new Date('2020-04-14T14:23:00'),
+            author: 'Пестова Анна',
+            likes: ['Бегун Юлий', 'Иван Иванов'],
+        },
+        {
+            id: '15',
+            description: 'я просто снова хочу ходить в универ и все успевать(',
+            createdAt: new Date('2020-04-14T14:24:00'),
+            author: 'Пестова Анна',
+            likes: ['Бегун Юлий', 'Иван Иванов'],
+        },
+        {
+            id: '16',
+            description: 'ладно, извините, меня стало слишком много в вашей ленте',
+            createdAt: new Date('2020-04-14T14:25:00'),
+            author: 'Пестова Анна',
+            likes: ['Бегун Юлий', 'Иван Иванов'],
+        },
+        {
+            id: '17',
+            description: 'я ботан',
+            createdAt: new Date('2020-04-14T14:26:00'),
+            author: 'Бегун Юлий',
+            likes: ['Бегун Юлий', 'Иван Иванов'],
+        },
+        {
+            id: '18',
+            description: 'юлий ботан',
+            createdAt: new Date('2020-04-14T14:27:00'),
+            author: 'Шикарный Макс',
+            likes: ['Бегун Юлий', 'Иван Иванов'],
+        },
+        {
+            id: '19',
+            description: 'университетские чебуреки лучшие!!!',
+            createdAt: new Date('2020-04-14T14:28:00'),
+            author: 'Гном Ник',
+            likes: ['Бегун Юлий', 'Иван Иванов'],
+        },
+        {
+            id: '20',
+            description: 'miss you :c',
+            createdAt: new Date('2020-04-14T14:28:30'),
+            author: 'Гном Ник',
+            likes: ['Бегун Юлий', 'Иван Иванов'],
+        },
+
+    ];
+    let empty={
+        id: '',
+        description: '',
+        createdAt: new Date(),
+        author: '',
+        likes: [],
+    };
+    let newPost={
+        id: '21',
+        description: 'ff',
+        createdAt: new Date('2020-03-14T14:28:30'),
+        author: 'l k',
+        likes: [],
+    };
+    let edPost={
+        description: 'ff',
+    };
+    function sortDate() {
+        return posts.sort((a,b)=> a.createdAt.getTime() > b.createdAt.getTime() ? -1 : 1)
+    }
+
+    function getPosts(skip, top, filterConfig){
+        sortDate();
+        let posts1 = posts.slice();
+        if (typeof filterConfig != "undefined") {
+            posts1 = posts1.filter(item => {
+                for (let key in filterConfig) if (key !== 'createdAt'  && filterConfig[key] !== item[key])
+                    return false;
+                return true;
+            });
+        }
+        return posts1.slice(skip , skip + top );
+    }
+
+    function getPost(id){
+        for (let i = 0; i < posts.length; i++) {
```

Comment:
Try to use array methods instead of manual iteration. [Find method docs](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/find)